Source/CarthageKit/Project.swift

```diff
@@ -458,6 +459,22 @@ public final class Project {
 				return copyProduct(dSYMURL, resolvedDestinationURL)
 			}
 	}
+	
+	/// Copies any *.bcsymbolmap files matching the given framework and contained
+	/// within the given directory URL to the directory that the framework
+	/// resides within.
+	///
+	/// If no bcsymbolmap files are found for the given framework, completes with
+	/// no values.
+	///
+	/// Sends the URLs of the bcsymbolmap files after copying.
+	public func copyBCSymbolMapsToBuildFolderForFramework(frameworkURL: NSURL, fromDirectoryURL directoryURL: NSURL) -> SignalProducer<NSURL, CarthageError> {
+		return BCSymbolMapsForFramework(frameworkURL, inDirectoryURL: directoryURL)
+			|> flatMap(.Merge) { source in
+				let target = frameworkURL.URLByDeletingLastPathComponent!.URLByAppendingPathComponent(source.lastPathComponent!)
+				return copyProduct(source, target.URLByResolvingSymlinksInPath!)
```

Comment:
Could you imitate the style of `copyDSYMToBuildFolderForFramework()` above for consistency and readability?